web/src/ferrostar-map.ts

```diff
@@ -297,7 +297,12 @@ export class FerrostarMap extends LitElement {
     });
 
     const responseData = new Uint8Array(await response.arrayBuffer());
-    return this.routeAdapter.parseResponse(responseData);
+    try {
+      return this.routeAdapter.parseResponse(responseData);
+    } catch (e) {
+      console.error("Error parsing route response:", String(e));
```

Comment:
I *think* it is more idiomatic JavaScript to write it like this... any thoughts @haysmike?

```suggestion
      console.error(`Error parsing route response: ${e}`);
```

Reply:
Thanks for the ping! Here's what I'd do:
```suggestion
      console.error("Error parsing route response:", e);
```

This should print out both the string and the error message nicely in the console and allow you to expand/collapse the stack.